student_class.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person_class'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, age, classroom, name = 'UNKNOWN', parent_permission: true)
+    super(id, name, age, parent_permission)
```

Comment:
- [ ] Kindly use the correct arrangement of parameters in the `super` method. `age` should come before `name`, and `parent_permission` should be assigned correctly.

  ```
  super(id, age, name, parent_permission: parent_permission)
  ```

Reply:
whoosh, parameter order mismatch. thanks